apps/ui/tsconfig.json

```
@@ -9,7 +9,9 @@
 		"jsx": "react-jsx",
 		"baseUrl": ".",
 		"paths": {
-			"@/assets/*": ["./assets/*"]
+			"@/components/*": ["./src/components/*"],
+			"@/assets/*": ["./src/assets/*"],
+			"@/pages/*": ["./src/pages/*"]
```

It's good practice to alphabetise lists of properties to improve searchability and prevent duplication
```suggestion
			"@/assets/*": ["./src/assets/*"],
			"@/components/*": ["./src/components/*"],
			"@/pages/*": ["./src/pages/*"]
```